micromouse/Motor.cpp

```diff
@@ -42,29 +123,51 @@ namespace Micromouse
 		{
 			digitalWrite(fwdPin, HIGH);
 			digitalWrite(bwdPin, LOW);
-			analogWrite(pwmPin, (int)(255 * speed));
+			setVoltage((int)(255 * speed), true);
```

Comment:
right so setVoltage takes in a speed and sets the voltage, it still follows the convention of -1 to 1 for speed and 0 to 255 for voltage. Then that way the code stays DRY